src/eventListeners/onImageRenderedBrushEventHandler.test.js

```diff
@@ -0,0 +1,510 @@
+import store from '../store/index.js';
+import {
+  getOutline,
+  renderOutline,
+  getRectsToFill,
+  renderFill,
```

Comment:
Will these ever be called by our consumer? These don't feel like public API, but I could be wrong.
I think the only thing we'd actually be testing is the handler itself, and leave these bits private?

Then you're just verifying that the render for the labelmap shows as expected.
Could use integration tests, JSDOM, or e2e if there's no good way to test this without a canvas implementation?

Reply:
Its true that a consumer would never call these directly, but I wanted to test the functionality of these pieces. Once we build e2e tests we can remove these as they are a bit brittle, as there isn't a good way to do this without a canvas implementation.